.typespec/realtime/client.tsp

```diff
@@ -1,85 +1,23 @@
-import "@azure-tools/typespec-client-generator-core";
-import "./models.tsp";
+import "@typespec/http";
 
-using Azure.ClientGenerator.Core;
-using OpenAI;
+import "./main.tsp";
+
+using TypeSpec.Http;
+
+/** The OpenAI REST API. Please see https://platform.openai.com/docs/api-reference for more details. */
+@service({
+  title: "OpenAI API",
+  termsOfService: "https://openai.com/policies/terms-of-use",
+  contact: {
+    name: "OpenAI Support",
+    url: "https://help.openai.com",
+  },
+  license: {
+    name: "MIT",
+    url: "https://github.com/openai/openai-openapi/blob/master/LICENSE",
+  },
+})
+@server("wss://api.openai.com/v1", "OpenAI Endpoint")
+@useAuth(BearerAuth)
+namespace OpenAI;
```

Comment:
I think converting this client.tsp into a main.tsp is a little confusing. 

Would it work if we did the following?

1. Create a separate folder under the ".typespec" folder called ".java", where we put this file (but call it "main.tsp") and also the "tspconfig.yaml" file.
2. Rename the "dotnet.tsp" and "java.tsp" files to "client.dotnet.tsp" and "client.java.tsp".
3. Move the "client.dotnet.tsp" and "client.java.tsp" files under ".typespec/realtime" instead of ".typespec/realtime/custom".
4. Revert the changes to the "/.typespec/realtime/main.tsp", but edit it to import both the "client.dotnet.tsp" file and "client.java.tsp" file.
5. Revert the changes to the "custom.tsp" file, and move it under the "custom" folder with the rest of the custom models.

Reply:
Thanks for all the feedback! Agreed in all accounts. Just regarding point 1 and after discussing with @trrwilson, I think that instead of having a cut "per language" we might actually prefer to have a cut "per artifact" we are hoping to generate from the spec. 

I will go ahead and create a separate folder where we can start defining entry points as we need them. For now, we generally need:
- Catch all entry point (for .NET and OpenAPIv3)
- Realtime specific (Java) 

Let me know what you think about this approach.

Reply:
Regarding moving `custom.tsp`, I didn't really do anything other than adding a couple of imports and removing an unused import which was breaking code gen in the java repo. Would it be okay to omit this change in this PR, as I would like to focus on the multiple entry point part of it and the Java support for realtime instead. More than happy to follow up in a different PR with any other restructuring.